.github/workflows/applications.yml

```diff
@@ -74,6 +74,9 @@ jobs:
       run: choco install -y -r awscli
 
     # Haskell Setup
+    - name: Set permissions for .ghcup
+      if: contains(matrix.os, 'ubuntu-*')
```

Comment:
It seems that this only fixed the Mac build, which comes as a surprise ...

Reply:
yeah i guess i can't use wildcards